01-main/packages/spotify-client

```diff
@@ -1,5 +1,5 @@
 DEFVER=2
```

Comment:
```suggestion
DEFVER=3
```
We need to update this (see [EXTREPO.md](https://github.com/wimpysworld/deb-get/blob/main/EXTREPO.md) ) so that anyone with a current install via `deb-get` will be prompted to re-install, which should remove the old key and replace it with the new one.